tests/integrationv2/providers.py

```diff
@@ -187,6 +187,9 @@ def setup_server(self):
         """
         cmd_line = ['s2nd', '-X', '--self-service-blinding', '--non-blocking']
 
+        # This is the last thing printed by s2nd before it is ready to begin
+        self.ready_to_test_marker = 's2n is ready'
```

Comment:
No, because the `_init_` method would be for both client and server. As for server we need to update only `ready_to_test_marker` and for client only `ready_to_send_input_marker`.

Reply:
What happens if we set ready_to_send_input_marker for server or ready_to_test_marker for client?